build.sh

```diff
@@ -5,6 +5,7 @@ make
 popd
 # f469 board
 pushd f469-disco/micropython/ports/stm32
-make BOARD=STM32F469DISC USER_C_MODULES=../../../usermods FROZEN_MANIFEST=../../../../manifest.py
+make BOARD=STM32F469DISC USER_C_MODULES=../../../usermods FROZEN_MANIFEST=../../../../manifest.py \
+     CFLAGS_EXTRA=-DMICROPY_PY_UCRYPTOLIB_CONSTS
```

Comment:
Do we adapt all the build scripts in submodules or should we copy paste them here to this repo with this minor change?